config/7.9.0/envs/rhpam-production-immutable.yaml

```diff
@@ -1,3 +1,7 @@
+#[[if .Console.Omit]]
+console:
+  omit: true
+#[[end]]
```

Comment:
If we want the console omission to depend on number of replicas set, maybe adding this to the yaml templates doesn't make sense?

Reply:
is this required for it to work?

Reply:
like the yaml ig gt Console.Replicas?

Reply:
i actually meant the go code you are working on in this PR that checks for replicas.

Reply:
updated it, wdyt?

Reply:
hmm... where'd the go code go? my thinking was get rid of this yaml addition and instead handle the omission logic in go.

Reply:
if the plan is to remove these yaml, I do agree to keep the logic on the go side.
let me edit it.

Reply:
Updated.